raiden-dapp/src/assets/logo_blue.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Don't we already have this logo in black? Wouldn't be better if you `css filter` to change color instead of adding a new asset?

Reply:
We do, I could not find a way of changing it to that specific blue color via css filter alone. But maybe I can access the SVG itself without any external library?

Reply:
Fixed it with filters! :)